coursebuilder/modules/explorer/_static/components/pages/homepage.html

```diff
@@ -73,85 +73,72 @@ <h2 class="section-label">An award winning platform</h2>
                 </div>
             </div>
 
-            <!--  How it works  -->
-            <div class="section how-it-works">
-                <div class="carousel-section-wrapper">
-                    <h2 class="section-label">How It Works</h2>
+          <!--  How it works  -->
+  <div class="section how-it-works">
```

Comment:
Is the indentation correct?

Reply:
@DavidLemayian All pages have been properly formatted. Python files are indented with 4 spaces (as recommend by pep 8) while html and javascript files are indented with 2 spaces.